### PR TITLE
On accepting mystery cube missions, sometimes give the player a more …

### PR DESCRIPTION
**Pull request by jmathes**
On accepting mystery cube missions, sometimes give the player a more ominous message. Ideally this will make the bounty hunter pursuit less confusing.

### CODE REVIEW

File: data/human/deep jobs.txt [hunk 1]
```diff
@@ -25,7 +25,16 @@ phrase "deep mystery cube pickup"
...
 		` `
 	word
-		`and pick up the mystery package. It is a small black cube, about two centimeters to a side, surprisingly heavy for its size, with no markings of any kind. Odd.`
+		`and pick up the mystery package. It is a small black cube, about two centimeters to a side, surprisingly heavy for its size, with no markings of any kind.`
+	word
+		` `
+	word
+		`Odd.`
+		`Strange.`
+		`Bizarre.`
```
Comment (Pointedstick) [inline data/human/deep jobs.txt:34]: I'd remove this one as it's a stronger word than "odd" or "strange", and also including it reduced the player's chance of seeing any of the new messages you've added .

**Comment by Pointedstick** [comment]
Good idea. This makes sense overall to me; just one suggestion for improvement.

File: data/human/deep jobs.txt (hunk 1 above)
Comment (tehhowch) [reply data/human/deep jobs.txt:34]: I'd maybe replace it with e.g. `Curious.` / `How curious.`.

IMO it is fine that the new text only shows up half the time--that might even be too much.

We could also double-weight the remarks, e.g. 
```
`Odd.`
`Odd.`
`Strange.`
`Strange.`
<...new flavor warnings>
```
to have them be just a touch more common, in aggregate, than the warnings.

**Review by Pointedstick — APPROVED** [review]
LGTM.

Anyone object? @Amazinite?

**Comment by Pointedstick** [comment]
Got semi-approval in the chat. Merging!

Thanks for the contribution.

**Pull request merged**